addons/l10n_ar/demo/respinsc_demo.xml

```diff
@@ -36,6 +36,13 @@
         <value model="res.company" eval="obj().env.ref('l10n_ar.company_ri')"/>
     </function>
 
+    <!-- Products created before the company don't get the default tax set, by default, 
+    although the tests are expecting it... Hence we force it -->
+    <function model="product.template" name="_force_default_tax">
+        <value eval="[]"/>
```

Comment:
You can also use a `search` here to avoid the weird search in case there is no product, even if there should be no product

Reply:
mmhh yes probably it would be cleaner... :+1: